doc/decisions/0002-sign-commits-with-gpg.md

```diff
@@ -0,0 +1,44 @@
+# 2. Sign Commits With GPG
+
+| Date       | Tags              |
+|------------|-------------------|
+| 2018-02-16 | process, security |
+
+
+## Status
+
+Proposed
+
+## Context
```

Comment:
I think there have been sufficient explanations why GPG signing is generally a good idea in #285, even if currently not implemented in an appropriate way. 

I agree that it played a role in this decision though so I'll mention it. It's not *the reason* however.

Reply:
The only reason that I understand is the status checks. The rest I couldn't, even reading the whole discussion and links over there. Besides, some gaps were left there, like the GitHub interface.

I'm okay with using it, but I just don't understand it with that discussion.

It's nice os uf to worry about security, but writing the goal of a decision without fully understand what is going on there is probably deceiving ourselves.

So I personally don't agree with being part of exposing like that the decision, since it's not true, at least to me.

Reply:
I added paragraphs to Context/Decision about the status check.

If there are still gaps please consider reading more about GPG signing, there are many resources out there. As stated in the conversation the benefits are only partially useful right now since we don't have a trusted method of exchanging public keys. As this method is provided by Status, GPG signing will allow commit author verification regardless of keys stored on GitHub.

Reply:
I think we need to have GPG signing @pablodip. Seems like that was the general consensus in #285 and it is 'company policy' per se. If you read the links, I can't add much more.

We could always simply re-use the `merge-pr.sh` script in `status-react` - it works well for me and is easy enough imo.